README.md

```diff
@@ -161,5 +161,4 @@ Give this project a star if you like it !
 
 This project is [MIT](./LICENSE.txt) licensed.
 
-
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
NOTE: This is only optional because it was not mentioned by the previous reviewer. However, I highly recommend that you take it into consideration.

[OPTIONAL] Here are key points to consider when documenting your README.

- The template logo should be removed. Optionally, the Microverse logo can be replaced with your own project logo if you have one. ✔️ 
- The only optional section is the FAQ section. If there's no information in the FAQ section then it should be deleted. ✔️ 
***However, since you documented the FAQ section, it would be best if you also added it to the table of contents.***
- All main headers from the [README template](https://github.com/microverseinc/readme-template) should be present. Even if there's no information for a section or sub-section, rather leave texts like "NA", "None" or "coming soon" than delete it.✖️ 
***You are missing the `Key Features` section which should have [1-3] key features listed and the `Acknowledgements` section. That includes the headings that are removed from the Table of contents. Only the FAQ section is optional***
- Future Features section should include 1-3 potential features to be added. ✔️ 
-  Each section description should be removed and replaced with their custom content. The section descriptions are meant to guide you in documenting your README. They are not meant to be part of the final documentation. ✖️ 
***The Future Features description is not removed***
For example:
![image](https://user-images.githubusercontent.com/38088818/216551345-4cb3cf19-9d8d-47d5-95be-ac8d47de0049.png)
![image](https://user-images.githubusercontent.com/38088818/216551482-2a274f75-4946-434a-97ea-746434b0a464.png)
![image](https://user-images.githubusercontent.com/38088818/216551578-aae6a149-69c8-4a13-bb22-4639defb421c.png)

- The License suggestion should be removed. ✔️ 
- There should not be any blank sections that do not have content. ✔️
- There should not be any broken links in the README file. ✔️ 
- Delete the live demo section if the application is not deployed. If project requirements say to deploy the project, the section should be restored and the link updated. ✔️